http/parse.go

```diff
@@ -28,12 +28,19 @@ func parseRequest(r *http.Request, root *cmds.Command) (*cmds.Request, error) {
 		getPath    = pth[:len(pth)-1]
 	)
 
-	cmd, err := root.Get(getPath)
+	cmdPath, err := root.Resolve(getPath)
 	if err != nil {
 		// 404 if there is no command at that path
 		return nil, ErrNotFound
 	}
 
+	for _, c := range cmdPath {
+		if c != nil && c.NoRemote {
```

Comment:
I don't think we need the nil check here. `getPath` is the path up to but not including the last element.

We need the nil check below because the last element may be either a value or a subcommand. However, none of the elements of `cmdPath` should be nil (`Resolve` returns an error if it fails to find a subcommand).

Reply:
> Resolve returns an error if it fails to find a subcommand

Yep, I'll remove this.